src/timely-util/src/containers.rs

```diff
@@ -126,12 +126,13 @@ mod container {
                 Column::Bytes(b) => {
                     assert_eq!(b.len() % 8, 0);
                     let mut alloc: Region<u64> = super::alloc_aligned_zeroed(b.len() / 8);
-                    bytemuck::cast_slice_mut(&mut alloc[..]).copy_from_slice(&b[..]);
+                    let alloc_bytes = bytemuck::cast_slice_mut(&mut alloc);
+                    alloc_bytes[..b.len()].copy_from_slice(b);
```

Comment:
This shouldn't be necessary, right? `alloc_bytes` should have capacity for all of `b`, so it should copy the whole of `b`, even if it has more capacity.

Reply:
`copy_from_slice` panics if the slices have different lengths ([docs](https://doc.rust-lang.org/std/primitive.slice.html#method.copy_from_slice)). Since `alloc_bytes` can be longer than `b`, we need to truncate it to avoid the panic.